internal/wiz/structs.go

```diff
@@ -244,6 +244,26 @@ type SAMLGroupMappingCreateInput struct {
 	Projects        []string `json:"projects"`
 }
 
+// SAMLGroupDetailsInput struct
+// Incomplete because 'description' field is missing as in the schema
+type SAMLGroupDetailsInput struct {
+	ProviderGroupID string   `json:"providerGroupId"`
+	Role            string   `json:"role"`
+	Projects        []string `json:"projects"`
+}
+
+// ModifySAMLGroupMappingPatch struct
+type ModifySAMLGroupMappingPatch struct {
+	Upsert *SAMLGroupDetailsInput `json:"upsert,omitempty"`
```

Comment:
I believe this schema for Upsert should take a slice of SamlGroupDetailsInput
![image](https://github.com/user-attachments/assets/41319c7a-2cdb-4422-bbd5-aada649167a9)


Then for update and delete ops, we should be able to do this in a single API call not once per group mapping